lexicons/app/bsky/feed/post.json

```diff
@@ -54,7 +54,7 @@
     },
     "textSlice": {
       "type": "object",
-      "description": "A text segment. Start is inclusive, end is exclusive.",
+      "description": "A text segment. Start is inclusive, end is exclusive. Indices are for utf16-encoded strings.",
```

Comment:
maybe also add a `Deprecated` note on this?

Reply:
Done